CONTRIBUTING.md

```diff
@@ -142,42 +142,42 @@ you can do this faster with the `-n`/`--no-discover` option. For example:
 
 to run a module:
 ```
-tox -epy -- -n test_max_weight_matching
+nox -e test -- -n test_max_weight_matching
 ```
 or to run the same module by path:
 ```
-tox -epy -- -n graph/test_nodes.py
+nox -e test -- -n graph/test_nodes.py
 ```
 to run a class:
 ```
-tox -epy -- -n graph.test_nodes.TestNodes
+nox -e test -- -n graph.test_nodes.TestNodes
 ```
 to run a method:
 ```
-tox -epy -- -n graph.test_nodes.TestNodes.test_no_nodes
+nox -e test -- -n graph.test_nodes.TestNodes.test_no_nodes
 ```
 
-It's important to note that tox will be running from the `tests/` directory in
+It's important to note that Nox will be running from the `tests/` directory in
 the repo, so any paths you pass to the test runner via path need to be relative
 to that directory.
 
 #### Visualization Tests
 
 When running the visualization tests, each test will generate a visualization
 and only fail if an exception is raised by the call. Each test saves the output
-image to the current working directory (which if running tests with `tox` is
+image to the current working directory (which if running tests with `nox` is
 `tests/`) to ensure the generated image is usable. However to not clutter the
 system each test cleans up this generated image and by default a test run does
 not include any way to view the images from the visualization tests.
 
 If you want to inspect the output from the visualization tests (which is common
 if you're working on visualizations) you can set the
-`RETWORKX_TEST_PRESERVE_IMAGES` environment variable to any value and this will
+`RUSTWORKX_TEST_PRESERVE_IMAGES` environment variable to any value and this will
```

Comment:
Did we update the env variables in the tests to reflect this and just forgot to update the docs?

Reply:
I'll send it in a separate PR